ddtrace/bootstrap/sitecustomize.py

```diff
@@ -72,6 +72,10 @@ def update_patched_modules():
     hostname = os.environ.get("DD_AGENT_HOST", os.environ.get("DATADOG_TRACE_AGENT_HOSTNAME"))
     port = os.environ.get("DATADOG_TRACE_AGENT_PORT")
     priority_sampling = os.environ.get("DATADOG_PRIORITY_SAMPLING")
+    profiling = asbool(os.environ.get("DD_PROFILING_ENABLED", False))
```

Comment:
Would it make sense to abide by this flag in the profiler itself? With this change someone could have `DD_PROFILING_ENABLED=false` and use `ddtrace.profiling.auto` manually and still have the profiler run.

Reply:
I don't think the tracer abide by `DATADOG_TRACE_ENABLED`, so I implemented the same behavior for consistency.

Reply:
Ugh yeah it's always bothered me that it doesn't :|. IMO it makes sense to abide by it but consistency also makes sense. 🤷‍♂️